### PR TITLE
Allow specific movies to be retrieved via query strings

### PR DESCRIPTION
**Pull request by willresen**
In this PR:

- Add `id` property to Movie model
- Add query string retrieval to React component
- Update `README.md`

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -15,14 +15,16 @@ class App extends React.Component {
   }
 
   componentDidMount() {
-    this.getFeaturedMovie('feature=true');
+    let qs = new URLSearchParams(window.location.search);
+    this.getFeaturedMovie(qs.get('id'));
```
Comment (HBMattoon) [inline client/src/App.jsx:19]: might want to set a default option in case the user doesn't add a query string

**Review by HBMattoon — APPROVED** [review]
looks great!

File: client/src/App.jsx (hunk 1 above)
Comment (glkapitan) [reply client/src/App.jsx:19]: Will, i'm not sure why I'm getting this error running my proxy.  FetchError: invalid json response body at http://localhost:3002/api/featured?id=1 reason: Unexpected end of JSON input

Comment (willresen) [reply client/src/App.jsx:19]: @glkapitan You may want to add some `console.log`s to see what data you are actually receiving - you are likely receiving an empty request or data that is not JSON.

**Pull request merged**